packages/cli/e2e/__tests__/fixtures/test-project/src/__checks__/homepage.test.ts

```diff
@@ -0,0 +1,16 @@
+// @ts-ignore
```

Comment:
Nit:

Would it be possible to move this test to be a unit test in `check-parser.spec.ts` as a separate unit test case and explicitly asserting that `external.first.page.js` and `external.second.page` are correctly picked up as dependencies?

The e2e tests are a bit heavier weight, so I don't typically run them locally. Having this covered as a unit test would make it a bit easier to run. I think that having the explicit assertion will also help make sure that we don't accidentally break the test.

Reply:
@clample , IMO, I think it's better to keep it in the E2E. Having a `test-project` with more complexity helps us to identify errors during the `test:e2e`. The previous `test-project` wasn't covering the 'missing extension' issue neither the 'duplicated resource' in Windows.
However, let me add a unit test to cover the file without extension dependencies pick up.

Reply:
Thanks @nahuelon, that sounds good to me

Reply:
@clample , done.